crates/wasmedge-sys/src/executor.rs

```diff
@@ -402,7 +402,8 @@ impl Executor {
                 )));
             }
             let mut value: libc::itimerspec = std::mem::zeroed();
-            value.it_value.tv_sec = timeout as i64;
+            value.it_value.tv_sec = timeout.as_secs() as _;
+            value.it_value.tv_nsec = timeout.subsec_nanos() as _;
```

Comment:
For better readability, please specify the concrete type used by `as` operator. Thanks a lot!